Source/Makefile

```diff
@@ -36,10 +33,14 @@ API_FILES := \
 	zzVersionChecker.cs \
 	ICostModifier.cs \
 	VectorUtils.cs \
+	KSPUtils.cs \
+	UnityUtils.cs \
+	EngineWrapper.cs \
+	MathUtils.cs \
 	$e
 
 RESGEN2	:= resgen2
-GMCS	:= gmcs
```

Comment:
Why are you changing gmcs to mcs?

Reply:
So Mono 2.11 unified them. If this is the case, replace the GMCS variable with MCS as well. `GMCS := gmcs` makes sense but `GMCS := mcs` does not.

Reply:
In addition, it would be good to add `-langversion:ISO-2 -sdk:2` after ${GMCS} to make sure we target the same version as `gmcs`.

Reply:
Nice thought, but it won't compile because there's lambda expressions and other language features used not in the C# 2.0 specification.